### PR TITLE
Update Breakfast readme.md

### PR DESCRIPTION
**Pull request by nyhdo93**
## 🎯 Objective
Hi team. Making this pull request to start laying down the groundwork for the "Breakfast" recipe. Please LMK if this looks good!

## 🛠️ Changes made
Updated all breakfast recipe sections aside from "Steps" which still needs to be added in from the Mushroom Omelette recipe. 

## 👥 Who needs to review this Pull Request? 

@frankzhang95 @amarisskim1 @eliasmufa

### CODE REVIEW

File: breakfast/readme.md [hunk 1]
```diff
@@ -1,20 +1,35 @@
-# Replace with recipe name
+# Mushroom Omelete With Chives
```
Comment (frankzhang95) [inline breakfast/readme.md]: 🍄 🍳 🥚

Comment (nyhdo93) [reply breakfast/readme.md]: Thanks Frank, will add in the emojis

File: breakfast/readme.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# Mushroom Omelete With Chives
+# Mushroom Omelete With Chives 🍄 🍳 🥚
```
Comment (amarisskim1) [inline breakfast/readme.md]: Change Egg with chive 🥚 --> 🌿

Comment (nyhdo93) [reply breakfast/readme.md]: Looks good

**Review by amarisskim1 — APPROVED** [review]
Looks great

**Review by amarisskim1 — APPROVED** [review]

**Review by nyhdo93 — COMMENTED** [review]
Looks good!

**Review by amarisskim1 — APPROVED** [review]

**Pull request merged**